vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -2,150 +2,181 @@
 
-## Build
+- `--hub-url`: Use this option to specify a custom Entando Hub, else ent defaults to the Entando Cloud Hub
+
+### Build
  
 | Command| Descriptions
 |:--|:--
-|`ent bundle build [component-name]` | Build a single component
-|`ent bundle build [mfe-1] [ms-2]...`| Build one or more named components
-|`ent bundle build --all`|Build all the components in the project
-|`ent bundle build --all-ms`|Build all microservices|
-|`ent bundle build --all-mfe`|Build all the micro frontends|
+|`ent bundle build [component-name]` | Build a single component |
+|`ent bundle build [mfe-1] [ms-2]...`| Build one or more named components passed as arguments |
+|`ent bundle build --all`| Build all the components in the project |
+|`ent bundle build --all-ms`| Build all the microservices |
+|`ent bundle build --all-mfe`| Build all the micro frontends |
 
 #### Command Details
-`ent bundle build`: Constructs the project files based on the provided `entando.json`. It executes according to the type of stack your components are built with. For instance, a React MFE starts an npm build process. A `build` log file is generated for each component inside the .entando/logs directory for your project.
+- The `build` command constructs the project files based on the `entando.json` descriptor
+
+- Component type and stack determines the build process, e.g. a React MFE executes an npm build
+
+- All the components in the bundle are built in parallel
+
+- A log file for each component is generated inside the .entando/logs directory of the project
 
-## Run	
+### Run	
 | Command| Descriptions
 |:--|:--
-|`ent bundle run [component-name]` | Locally run your single component
-|`ent bundle run [mfe-1] [ms-2]...`| Locally run one or more named components passed as arguments 
-|`ent bundle run --all`| Locally run all the components in the bundle
-|`ent bundle run --all-ms`| Locally run the microservices in the bundle
-|`ent bundle run --all-mfe`| Locally run the MFEs in the bundle
+|`ent bundle run [component-name]` | Locally run a single component |
+|`ent bundle run [mfe-1] [ms-2]...`| Locally run one or more named components passed as arguments |
+|`ent bundle run --all`| Locally run all the components in the project |
+|`ent bundle run --all-ms`| Locally run all the microservices |
+|`ent bundle run --all-mfe`| Locally run all the micro frontends |
 
 #### Command Details
-`ent bundle run`: Executes processes dependent on the component type and stack. For example, `mvn spring-boot:run` will execute for a Spring Boot microservice. All the components in the bundle will run in parallel, with the logs printed to the standard output.
+- The `run` command executes processes in accordance with the component type and stack, e.g `mvn spring-boot:run` is executed for a Spring Boot microservice
```

Comment:
i really think this adds unnecessary and obvious verbiage but i added the commands back in and indented the existing bullet points. i didn't want to stick with one bullet per command because the details would get lost in a paragraph that would read haltingly without extra phrases to connect the concepts; instead, each point should have a dedicated bullet.